### PR TITLE
Makes dehydrated carps more fun

### PR DESCRIPTION
**Pull request by kilkun**
Gives the dehydrated carp a variable that contains the mob path that it spawns. So admins can var edit it for admin abuse. 20 dehydrated clown snpcs anyone?

simple pr. ignore pull history pls.

### CODE REVIEW

**Comment by nullbear** [comment]
use this: http://imgur.com/a/lfTVJ

**Comment by PKPenguin321** [comment]
or just base your branch off of upstream/master and don't commit until you actually have compiling code

File: code/game/objects/items/dehy_carp.dm [hunk 1]
```diff
@@ -36,7 +37,7 @@
 	//Wait for animation to end
 	sleep(6)
 	//Make space carp
-	var/mob/living/simple_animal/hostile/carp/C = new /mob/living/simple_animal/hostile/carp(get_turf(src))
+	var/mobtype/C = new mobtype(get_turf(src))
```
Comment (Core0verload) [inline code/game/objects/items/dehy_carp.dm]: this is where it fails

Comment (PKPenguin321) [inline code/game/objects/items/dehy_carp.dm]: oh yeah huh
i guess you would just remove the var definition and just have "new mobtype..." stuff, then replace mentions of the var C with "mobtype"

**Comment by kilkun** [comment]
I see. Still learning the kinks of byond declaration. Didn't give me an error when I did a test compile...

File: code/game/objects/items/dehy_carp.dm [hunk 2]
```diff
@@ -36,16 +37,16 @@
 	//Wait for animation to end
 	sleep(6)
 	//Make space carp
-	var/mob/living/simple_animal/hostile/carp/C = new /mob/living/simple_animal/hostile/carp(get_turf(src))
+	new mobtype(get_turf(src))
```
Comment (RemieRichards) [inline code/game/objects/items/dehy_carp.dm]: ```var/mob/living/L = new motype(get_turf(src))```
then change all ```mobtype``` to ```L``` (EXCEPT the var definition)

**Comment by coiax** [comment]
Don't worry, multiple commits just shows you're enthusiastic.

**Pull request merged**